tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -644,45 +644,6 @@ pfcwd/test_pfc_config.py::TestPfcConfig::test_forward_action_cfg:
      conditions:
         - "asic_type in ['cisco-8000']"
 
-#######################################
-#####         platform_tests      #####
-#######################################
-platform_tests/api/test_sfp.py::TestSfpApi::test_reset:
-  skip:
-    reason: "platform does not support sfp reset"
-    conditions:
-      - "platform in ['x86_64-cel_e1031-r0']"
-
-platform_tests/api/test_sfp.py::TestSfpApi::test_tx_disable_channel:
-  skip:
-    reason: "platform does not support"
-    conditions:
-      - "platform in ['x86_64-cel_e1031-r0']"
-
```

Comment:
These checks are not for cisco platfrom, are these platform getting skipped in checks in [tests_mark_conditions_platform_tests.yaml](https://github.com/sonic-net/sonic-mgmt/pull/7771/files#diff-25d358a45bebd38dae8a3812fce5e9d01ee3efb91d3f5b7328aa36c8b7ae1ee9)

Reply:
@arlakshm , I moved this from the other conditional marker to the platform one because this portion belongs to platform conditinal marker and would be easily forgoten if not moved.  all platform related conditional check should reside in one single conditional marker.  It has nothing to do with the Cisco check to skip for test_reset but is an effort to clean it up and make it less likely to cause error and confusions when condition check are kept on two places...